recipes/tasklogger/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "tasklogger" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 50a3a20c9d675dc0e40f8a7d532bf845526ffa98a21ed0cd5ed957fcd40e54c1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - future
+    - pip
+    - python
+  run:
+    - future
+    - python
+
+test:
+  imports:
+    - tasklogger
+
+about:
+  home: https://github.com/scottgigante/tasklogger
+  license: GNU General Public Version 2
+  license_family: GPL2
+  license_file: LICENSE
```

Comment:
The `license_file` isn't included in the source distribution, so please package it manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

I see that you put in a PR upstream (https://github.com/scottgigante/tasklogger/pull/2). Please add a link to the PR as a comment in the `meta.yaml` as a reminder to check if it's been fixed for future releases